packages/embark-blockchain-listener/src/dev_txs.js

```diff
@@ -8,20 +8,20 @@ class DevTxs {
     this.networkId = null;
     if (options.provider) {
       this.provider = options.provider;
-    } else if (this.blockchainConfig.wsRPC) {
+    } else if (this.blockchainConfig.wsRPC !== false) {
       this.provider = new Web3.providers.WebsocketProvider(
         buildUrl(
           'ws',
-          this.blockchainConfig.wsHost,
-          this.blockchainConfig.wsPort
+          dockerHostSwap(this.blockchainConfig.wsHost) || defaultHost,
+          this.blockchainConfig.wsPort || 8546
```

Comment:
We should put this default port in a const and reuse it.

Reply:
Will do.

Reply:
Done.